src/ProjectTemplates/Web.ProjectTemplates/content/Grpc-Application/Grpc-Application.Server/Program.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
```

Comment:
I removed some code here so some of these using statements are now redundant. Will remove.